Jenkinsfile

```diff
@@ -1,11 +1,11 @@
-@Library('global-jenkins-library@1.3.0') _
+@Library('global-jenkins-library@1.8.0') _
 buildJavaProject(
+        buildInfo: getBuildInfo(),
         integrationTestsEnvVars: [],
         shouldPublishJars: true,
         shouldPublishDockerImages: true,
         dockerfileDir: '.',
-        //dockerfileFilename: "Dockerfile-local",
-        buildContext: '.',
```

Comment:
why did you remove this configuration which is kept in the other PRs ?

Reply:
Current implementation of jenkins-library is awkward: If `buidContext` is not set, default `buildContext` will be `dockerfileDir` (whose default is `docker/`). In next PRs, it might be best to remove that rule and set all defaults to `.`.

Reply:
I agree I find this part awkward too and to work on this in a next development.